schema/validator/bool.go

```diff
@@ -39,8 +39,8 @@ type BoolRequest struct {
 
 // BoolResponse is a response to a BoolRequest.
 type BoolResponse struct {
-	// Diagnostics report errors or warnings related to validating the data
-	// source configuration. An empty slice indicates success, with no warnings
+	// Diagnostics report errors or warnings related to validating the data source, provider, or resource
```

Comment:
Nit: I know its so tempting to fix these all at once, but definitely feel free to create followup issues/PRs too with large PRs to reduce review burden 😉

Reply:
This was selfishly for me lol 🤣

Opened #958, when it merges will remove all the diffs here 👍🏻